week2-assignment/index.html

```diff
@@ -0,0 +1,140 @@
+          <select id="role">
+            <option value="" hidden selected>OB/YB 선택</option>
+            <option value="OB">OB</option>
+            <option value="YB">YB</option>
+          </select>
+        </div>
+    
+        <div class="filter-item">
+          <label for="week1">1주차 금잔디조</label>
+          <input type="text" id="week1" placeholder="1주차 금잔디조를 입력하세요 (1~9)">
+        </div>
+
+        <div class="filter-item">
+          <label for="week2">2주차 금잔디조</label>
+          <input type="text" id="week2" placeholder="2주차 금잔디조를 입력하세요 (1~9)">
+        </div>
+      </div>
+    
+      <div class="button-group">
+        <button type="reset" id="reset-button">초기화</button>
+        <button type="submit" id="search-button">검색</button>
+      </div>
+    </section>
+
+
+    <section class="table">
+      <div class="button-table">
+        <button id="delete-selected">선택 삭제</button>
+        <button id="add-member">추가</button>
+      </div>
+    
+      <table id="members-table">
+        <thead>
+          <tr>
+            <th><input type="checkbox" id="select-all"></th>
+            <th>이름</th>
+            <th>영문 이름</th>
+            <th>GitHub</th>
+            <th>성별</th>
+            <th>역할</th>
+            <th>1주차 금잔디</th>
+            <th>2주차 금잔디</th>
+          </tr>
+        </thead>
+        <tbody id="members-tbody">
+
+        </tbody>
+      </table>
+    </section>
+  </main>
+
+  <div class="modal" id="modal">
+    <div class="modal-content">
+      <span class="close" aria-label="닫기">&times;</span>
+      <h2>새 멤버 추가</h2>
+      <form id="add-member-form">
+        <label for="modal-name">이름</label>
+        <input type="text" id="modal-name" name="name" placeholder="이름을 입력하세요" required>
+
+        <label for="modal-eng-name">영문 이름</label>
+        <input type="text" id="modal-eng-name" name="engName" placeholder="영문 이름을 입력하세요" required>
+
+        <label for="modal-github">GitHub ID</label>
+        <input type="text" id="modal-github" name="github" placeholder="GitHub ID를 입력하세요" required>
+
+        <label for="modal-gender">성별</label>
+        <select id="modal-gender" name="gender" required>
+          <option value="" hidden selected>성별 선택</option>
+          <option value="male">남자</option>
+          <option value="female">여자</option>
+        </select>
+
+        <label for="modal-role">역할</label>
+        <select id="modal-role" name="role" required>
+          <option value="" hidden selected>OB/YB 선택</option>
+          <option value="OB">OB</option>
+          <option value="YB">YB</option>
+        </select>
+
+        <label for="modal-week1">1주차 금잔디조</label>
+        <input type="text" id="modal-week1" name="week1" placeholder="1주차 금잔디조를 입력하세요 (1~9)" required>
+
+        <label for="modal-week2">2주차 금잔디조</label>
+        <input type="text" id="modal-week2" name="week2" placeholder="2주차 금잔디조를 입력하세요 (1~9)" required>
+
+        <button type="button" id="add-member-button">추가</button>

```

Comment:
p3: 해당 부분은 폼 내의 버튼으로 데이터를 제출하기 위해 다음과 같이 `type="submit"`으로 수정해주는 것이 어떨까요?
```suggestion
        <button type="submit" id="add-member-button">추가</button>
```